.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.17
+          go-version: 1.19
       - run: GOPROXY=https://proxy.golang.org,direct GOSUMDB=off go get -u golang.org/x/lint/golint; go list ./kuma/... | grep -v /vendor/ | xargs -L1 /home/runner/go/bin/golint -set_exit_status
```

Comment:
We will need to update this to `golangci-lint`

See https://github.com/meshery/meshery-istio/pull/490 for comparison.

Reply:
Makes much sense. Please do carry this PR forward, if you're able to make commits on the PR/branch.